### ISSUE TITLE
build failure from bad variable conversion

### ISSUE THREAD

**Issue by quequotion** [elementary/switchboard-plug-onlineaccounts#41]
https://github.com/elementary/switchboard-plug-onlineaccounts/blob/369e4703f31306d68194b58e6b30959910579539/src/Views/ACListBox.vala#L177

This came up while building the package:

    src/Views/ACListBox.vala:177.47-177.49: error: Argument 1: Cannot convert from `unowned GLib.List<Signon.SecurityContext>' to `unowned Signon.SecurityContextList'
                    info.set_access_control_list (acl);
                                                  ^^^

Might be that I have a different version of glib or gsignond that what is expected (Archlinux).

**Comment by tintou** [comment]
Yeah, these has been recent API changes on the Signond side, we don't support the very latest version right now

**Comment by quequotion** [comment]
Any idea what I could to to patch around for arch in the mean time?

**Comment by worldofpeace** [comment]
@quequotion @tintou I can not longer produce this using [gsignond](https://gitlab.com/accounts-sso/gsignond/commit/477b96ae64d7bc7e7058b73d043d23e8569533b0) and [libgsignon-glib](d1bae907664e3775835b96a191165963c1583886) versions
